doc/scheduler-scripting.tex

```diff
@@ -561,6 +561,55 @@ \subsubsection{Graphical Applications}
 	\label{fig:pycharm}
 \end{figure}
 
+This is an example of running Jupyter-Notebooks.
```

Comment:
We need a `\subsubsection{Jupyter Notebooks}` before this line, so it shows up in the TOC, and easily linkable in tickets and replies.